interface-definitions/protocols-pim.xml.in

```diff
@@ -41,6 +41,25 @@
                   </constraint>
                 </properties>
               </leafNode>
+              <leafNode name="bfd">
+                <properties>
+                  <help>BFD support</help>
+                  <completionHelp>
+                    <list>enable disable</list>
+                  </completionHelp>
```

Comment:
If we have “bfd” does it mean that we want to enable it? Let’s avoid enable/disable

Reply:
Please see BGP neighbors, there is a simple bfd node that is valueless

Reply:
All nodes with values like true/false, enable/disable that still remain in the system are legacy and should be converted to valueless as soon as it's feasible. The policy not to make any new ones has been in place since Vyatta times, in fact. ;)